selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -93,11 +96,24 @@ def parse_model(model_msg, model_error):
       v = np.zeros(len(T_IDXS_MPC))
       a = np.zeros(len(T_IDXS_MPC))
       j = np.zeros(len(T_IDXS_MPC))
-    return x, v, a, j
+    if len(model_msg.meta.disengagePredictions.gasPressProbs) > 1:
+      throttle_prob = model_msg.meta.disengagePredictions.gasPressProbs[1]
+    else:
+      throttle_prob = 1.0
+    return x, v, a, j, throttle_prob
 
   def update(self, sm):
     self.mpc.mode = 'blended' if sm['selfdriveState'].experimentalMode else 'acc'
 
+    if len(sm['liveCalibration'].rpyCalib) == 3 and len(sm['carControl'].orientationNED) == 3:
```

Comment:
Doesn't `carControl->orientationNED` already use `PoseCalibrator` in controlsd?

Reply:
I'll check, it looked _very_ far off in some of the logs I looked at though.

Reply:
Yes that should already be calibrated